### DISCUSSION TITLE
Errors running on Chromebook under Crostini

### DISCUSSION THREAD

**Discussion by bertradio** [kovidgoyal/kitty#6677] [General]
I have downloaded kitty from the aarch arm repositories and it works well. But it is version 0.19. When I download the latest version from the download site and try to run it, I get these errors:

./kitty: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./kitty)
./kitty: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./kitty)

What can I do? My system does have libc.so.6 but it is an earlier version.

**Comment by kovidgoyal** [comment]
You can build from source yourself, as described in
https://sw.kovidgoyal.net/kitty/build/ 

On Thu, Oct 05, 2023 at 04:48:15PM -0700, bertradio wrote:
> I have downloaded kitty from the aarch arm repositories and it works well. But it is version 0.19. When I download the latest version from the download site and try to run it, I get these errors:
> 
> ./kitty: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./kitty)
> ./kitty: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./kitty)
> 
> What can I do? My system does have libc.so.6 but it is an earlier version.
> 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6677
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Comment by bertradio** [comment]
Thanks so much for the speedy reply.

I am a bit of a newbee. I tried to compile but do not have the Go compiler on my system. I went to their site and I'm not sure which file to install.
